css/src/atomics/typography.scss

```diff
@@ -1,3 +1,15 @@
+// Font stack
+
+.is-monospace {
```

Comment:
These should match the same pattern as other atomics.

```
.font-family-monospace
.font-family-quote
```

Reply:
updated